src/bio.js

```diff
@@ -1,15 +1,15 @@
 function processData(response) {
-    const bio = response.bio;
-    const bio2 = response.bio2;
-    let output = "";
-    for(var i = 0; i < bio.length; i++){
-      output += bio[i];
-    }
-    for(let j = 0;j < bio2.length; j++){
-      output += bio2[j];
-    }
-     document.getElementById("bio").innerHTML = output;
-  }
+  const bio = response.bio;
+  const bio2 = response.bio2;
 
+  let listArray = document.getElementById("bio");
+  for (let i = 0; i < bio.length; i++) {
+    listArray.innerHTML = `${bio[0]}${bio[1]}${bio[2]}`;
```

Comment:
You're looping properly, but not actually using the index. Array indices should never be hard-coded when iterating over the array.  What you're doing here is basically:

`const arr = [ 1, 2, 3 ];`
`let str = arr[0] + arr[1] + arr[2];`
`// str is now '123'`

The point of iterating over an array instead of hard-coding it is to make your code dynamic.  If the data changed, your solution here would no longer work.

Imagine these scenarios:
`const arr = [ 1, 2 ];`
`let str = arr[0] + arr[1] + arr[2]; // This fails because there is no value at the array index 2 (arr[2] is invalid and will actually break the execution of the rest of the JS on the site)`
`const arr = [ 1, 2, 3, 4 ];`
`let str = arr[0] + arr[1] + arr[2];`
`// str is ONLY '123' because I never added index 3 (arr[3] = 4)`